wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -54,7 +54,7 @@
             </div>
 
             {% if not slim_sidebar_enabled %}
-            <div id="nav-toggle" class="nav-toggle icon text-replace">{% trans "Menu" %}</div>
+            <button id="nav-toggle" class="nav-toggle icon text-replace">{% trans "Menu" %}</button>
```

Comment:
suggestion:

```suggestion
            <button type="button" id="nav-toggle" class="nav-toggle icon text-replace">{% trans "Menu" %}</button>
```

reference: https://github.com/wagtail/wagtail/issues/6592#issuecomment-735226755

Reply:
Thank you for pointing out the missing type! Commit suggestion applied